src/app/app/app-effects/effects/cases.app.effects.spec.ts

```diff
@@ -70,7 +77,9 @@ describe('CasesAppEffects', () => {
 		}).compileComponents();
 	}));
 
-	beforeEach(inject([Store, casesConfig], (_store: Store<any>) => {
+	beforeEach(inject([ImageryCommunicatorService, Store, casesConfig, ],
```

Comment:
casesConfig, ] - > remove the last comma inside the brackets

Reply:
Done